message/validation/consensus_validation.go

```diff
@@ -396,3 +450,13 @@ func (mv *messageValidator) roundRobinProposer(height specqbft.Height, round spe
 	index := (firstRoundIndex + int(round) - int(specqbft.FirstRound)) % len(committee)
 	return committee[index]
 }
+
+func encodeOperators(operators []spectypes.OperatorID) ([]byte, error) {
+	buf := new(bytes.Buffer)
+	for _, operator := range operators {
+		if err := binary.Write(buf, binary.LittleEndian, operator); err != nil {
+			return nil, err
+		}
+	}
+	return buf.Bytes(), nil
```

Comment:
why not do something like a hash?

Reply:
The most common case is `8*4=32` bytes, the max possible size with 13 operators is `8*13=104` bytes. Sha256 is 32 bytes, so the difference won't be big, but I don't mind hashing this

Reply:
done